docs/se/config/advanced-configuration.adoc

```diff
@@ -790,7 +784,7 @@ link:{config-javadoc-base-url}/io/helidon/config/ConfigSources.CompositeBuilder.
 which extends `Config.Builder`.
 
 Because a composite source might yield more numerous change events -- because of the
-multiple underlying sources -- your application can specify a _debounce timeout_
+multiple underlying sources -- your application can specify _debounce timeout_
 for the composite source by invoking the `CompositeBuilder.changesDebounce(Duration)`
 method. The composite source aggregates multiple change events within this period
 into a single event and broadcasts that one instead and reloads the sources at
```

Comment:
unclear what is meant here: 
..... and broadcasts that one [event] instead and reloads the sources [that??] can specify a non-default...

Suggest making 2 sentences if possible. 
  The composite source aggregates multiple change events within this period into a single event and broadcasts that one instead. Next, it reloads the sources that can specify a non-default...

Reply:
Updated the statements for clarity.